rosdep/python.yaml

```
@@ -6104,6 +6104,12 @@ python3-boto3:
     '*': ['python%{python3_pkgversion}-boto3']
     '7': null
   ubuntu: [python3-boto3]
+python3-breathe:
+  debian: [python3-breathe]
+  fedora: [python3-breathe]
+  opensuse: [python3-breathe]
+  rhel: [python3-breathe]
```

This package isn't available in RHEL 7 so that has to be nulled out. I also updated this to use the python3_pkgversion macro for the rhel key which I believe helps us in the situation that the default version of python3 for a RHEL release isn't the version of python3 we're using for ROS on that platform but @cottsay should double check before I make a PR to the contributing and review guidelines.

```suggestion
  rhel:
    '*': ['python%{python3_pkgversion}-breathe']
    '7': null
```